ckanext/ontario_theme/templates/internal/datastore/snippets/dictionary_form.html

```diff
@@ -0,0 +1,34 @@
+{% import 'macros/form.html' as form %}
+
+<h3>{{ _( "Field {num}.").format(num=position) }} {{ field.id }} ({{ field.type }})</h3>
+
+{#
+  Data Dictionary fields may be added this snippet. New fields following
+  the 'info__' ~ position ~ '__namegoeshere' convention will be saved
+  as part of the "info" object on the column.
+#}
+
+{{ form.select('info__' ~ position ~ '__type_override',
+  label=_('Type Override'), options=[
+  {'name': '', 'value': ''},
+  {'name': 'array', 'value': 'int[]'},
+  {'name': 'array', 'value': 'text[]'},
+  {'name': 'boolean', 'value': 'boolean'},
+  {'name': 'date', 'value': 'date'},
+  {'name': 'float', 'value': 'float'},
```

Comment:
I checked how postgres stores values specified as float and it actually represents them as `double precision`. So can you  change `'value': 'float'` to `'value': 'double precision'` ?

![Screenshot 2023-08-15 at 11 28 08](https://github.com/ongov/ckanext-ontario_theme/assets/1254764/9c2eff0b-d723-4aa9-995a-9f07505be172)